hikvision_tftpd_test.py

```diff
@@ -78,14 +79,16 @@ def test_eaddrnotavail(self):
             self.fail('expected an error')
 
     def test_eaccess(self):
-        try:
-            # The test shouldn't be running as root.
-            hikvision_tftpd.Server(('127.0.0.1', 1), ('127.0.0.1', 3), '')
-        except hikvision_tftpd.Error, e:
-            self.assertTrue('permission' in e.message, 'Unexpected: %r' % e)
-        else:
-            self.fail('expected an error. '
-                      '(did you run the tests as root? don\'t.)')
+		# No check for root on Windows Platform
```

Comment:
This would be clearer if it marked the test as skipped rather than done (without doing anything) on Windows. There's a very similar example to do that in the Python unittest documentation:

https://docs.python.org/2/library/unittest.html
    @unittest.skipUnless(sys.platform.startswith("win"), "requires Windows")

so I think @unittest.skipIf(sys.platform.startswith('win'), 'irrelevant on Windows') would work.